### PR TITLE
Rowan/feature/hk tt

### PR DESCRIPTION
**Pull request by rrasmuss4200**
We now have serialized JSON housekeeping data. This is a preliminary version of it. File naming still needs to be worked on.

### CODE REVIEW

File: ex3_ground_station/cli_ground_station/src/main.rs [hunk 1]
```diff
@@ -80,13 +82,28 @@ fn build_msg_from_operator_input(operator_str: String) -> Result<Msg, std::io::E
...
+    if dest_id == DFGM && opcode == dfgm::GET_DFGM_DATA{
+        dest_id = ComponentIds::BulkMsgDispatcher as u8;
+        msg_type = MsgType::Bulk as u8;
+        // Works
+        msg_body = "../handlers/dfgm_handler/dfgm_data".as_bytes().to_vec();
+    } else if dest_id == IRIS && opcode == IRIS::DownlinkHK as u8{
+        dest_id = ComponentIds::BulkMsgDispatcher as u8;
+        msg_type = MsgType::Bulk as u8;
+        // Bulk Msg Disp returns NotFound Error: No such file or directroy when I have it there :(
+        msg_body = "../handlers/iris_handler/iris_data".as_bytes().to_vec();
```
Comment (dheadrick1618) [inline ex3_ground_station/cli_ground_station/src/main.rs]: Need to differentiate between payload data and housekeeping data - I think we should have a single folder for hk, payload data, and logs inside each handler.

File: ex3_ground_station/cli_ground_station/src/main.rs [hunk 2]
```diff
@@ -168,22 +185,23 @@ fn read_bulk_msgs(
...
     // ADD future dir names here depending on source
-    let dir_name = if src == DFGM {
-        "dfgm"
-    } else if src == 99 {
-        "test"
-    } else {
-        "misc"
+    let mut dir_name: String = match src {
+        DFGM => "dfgm".to_string(),
+        4 => "iris".to_string(), // IRIS ID
```
Comment (dheadrick1618) [inline ex3_ground_station/cli_ground_station/src/main.rs:187]: Use the enums for these instead of just the component ID int

**Review by dheadrick1618 — CHANGES_REQUESTED** [review]
Minor semantic and project data location hierarchy changes requested. Good work so far.

File: ex3_ground_station/cli_ground_station/src/main.rs (hunk 1 above)
Comment (rrasmuss4200) [reply ex3_ground_station/cli_ground_station/src/main.rs]: We can assign someone to do this

**Review by dheadrick1618 — APPROVED** [review]
LGTM great work :D

**Pull request merged**